dts/arm/renesas/ra/ra8/ra8x1.dtsi

```diff
@@ -527,6 +527,25 @@
 				status = "disabled";
 			};
 		};
+
```

Comment:
Please put dts and board related changes into separate commits. Your PR would have four commits, manifest change, dts changes, driver implementation, board-related changes.

Reply:
I fixed it